testing/internal/e2e/boundary/user.go

```diff
@@ -48,10 +47,9 @@ func CreateNewUserCli(t testing.TB, ctx context.Context, scopeId string) string
 	newUserId := newUserResult.Item.Id
 	t.Cleanup(func() {
 		AuthenticateAdminCli(t, context.Background())
-		output := e2e.RunCommand(ctx, "boundary",
+		e2e.RunCommand(ctx, "boundary",
```

Comment:
Since the test was deleting the user, this check was failing because the user was already deleted. I'm thinking about potentially removing these checks in any of the cleanup steps. Thoughts?

Reply:
Should we not do this on a case by case basis? Sure it makes sense to expect an error when trying to delete a user that was successfully deleted as part of the test, but you could just write this logic to ignore that error specifically and still error on other errors.

Reply:
Mm. I did think about that. I'll explore it further.

Reply:
Pushed up a change